modules/serviceregistry/src/test/java/org/opencastproject/serviceregistry/impl/ServiceRegistryJpaImplTest.java

```diff
@@ -350,9 +406,13 @@ public void testDispatchingJobsHigherMaxLoad() throws Exception {
     JobBarrier barrier = new JobBarrier(null, serviceRegistryJpaImpl, testJob);
     try {
       barrier.waitForJobs(2000);
+      //We should never successfully complete the job, so if we get here then something is wrong
       Assert.fail();
     } catch (Exception e) {
       testJob = serviceRegistryJpaImpl.getJob(testJob.getId());
+      //Some explanation here: If the load exceeds the global maximum node load (ie, jobLoad > all individual node max
+      // loads), then we dispatch to the biggest, even if it's not going to normally accept the job.  That node may still
+      // reject the job, but that's AbstractJobProducer's job, not the service registry's
       Assert.assertEquals(TEST_HOST_OTHER, testJob.getProcessingHost());
```

Comment:
@gregorydlogan Should this job really have a processing Host here while still in QUEUED state? This job is in QUEUE'd state because it was rejected by the host_other. What is the point of preserving the attempted  dispatch if that host will not take the job?

Reply:
The issue @karendolan refers to is explained in https://opencast.jira.com/browse/MH-13675 (see problem 2 there). I will provide a fix soon.